resources/conf/log_config.toml

```diff
@@ -67,3 +71,5 @@ reqTxDur = "%REQUEST_TX_DURATION%"
 respTxDur = "%RESPONSE_TX_DURATION%"
 reqDur = "%REQUEST_DURATION%"
 respDur = "%RESPONSE_DURATION%"
+apiUUID = "%DYNAMIC_METADATA(envoy.filters.http.ext_authz:apiUUID)%"
```

Comment:
Isn't it better to keep API UUID in the beginning ?

Reply:
Yeah, it is better. But the reason I changed the order was, in TEXT format we have appended the apiUUID at the end of the line. To have the same order I moved ApiUUID down.